.github/in-cluster-test-scripts/aks-install.sh

```diff
@@ -7,10 +7,6 @@ set -e
 cilium install \
   --version "${CILIUM_VERSION}" \
   --disable-check=az-binary \
-  --azure-subscription-id "${AZURE_SUBSCRIPTION_ID}" \
-  --azure-node-resource-group "${AZURE_NODE_RESOURCE_GROUP}" \
-  --azure-tenant-id "${AZURE_TENANT_ID}" \
-  --azure-client-id "${AZURE_CLIENT_ID}" \
-  --azure-client-secret "${AZURE_CLIENT_SECRET}" \
```

Comment:
Now that the `AZURE_*` env vars seem to be unused, I guess we can drop them here as well:

https://github.com/cilium/cilium-cli/blob/0c6a9fe9d026e9cbb870da19de283c88b5fbb814/.github/cilium-cli-test-job-chart/templates/job.yaml#L37-L46

and then also the corresponding values:

https://github.com/cilium/cilium-cli/blob/0c6a9fe9d026e9cbb870da19de283c88b5fbb814/.github/cilium-cli-test-job-chart/values.yaml#L16-L21

Reply:
I'd suggest we keep them in until we decide if we're actually dropping support for the Azure IPAM or not. As of now it hasn't been clearly stated and I think both BYOCNI and Azure IPAM are going to live alongside each other for a while, so I have hope that we can have CI for both at the same time.